app/components/pull-request-page.js

```diff
@@ -201,11 +207,12 @@ export default Ember.Component.extend({
     },
 
     submit() {
+      set(this, "triedToSubmit", true)
       if (Ember.isBlank(get(this, "model.submitterEmail")) || get(this, "model.submitterEmail") === "") {
         swal("Make sure your email is filled out!")
         return false
       }
-      if (!get(this, "errors")) {
+      if (Ember.isEmpty(get(this, "errors"))) {
```

Comment:
I can't add a comment on the area below, but I'd change the swal to something a bit more descriptive like, "These standards aren't ready for submitting, yet! Make sure all the checkboxes are green and fix any errors."